mslice/models/workspacemanager/mantid_workspace_provider.py

```diff
@@ -60,9 +74,6 @@ def _processLoadedWSLimits(self, workspace):
             else:
                 self._limits[ws_name] = None
```

Comment:
I know you didn't change this but could we use `self._limits[ws_name] = {}` here instead for consistency with other places here?